modules/tinymce/src/themes/silver/main/ts/ui/context/ContextUi.ts

```diff
@@ -28,6 +28,7 @@ interface ContextToolbarSpec {
   readonly sink: AlloyComponent;
   readonly onHide: () => void;
   readonly onBack: () => void;
+  readonly focusElement: (el: SugarElement<any>) => void;
```

Comment:
I think we can make this HTMLElement if you change the type of ChangeSlideEvent to that type. I think using just SugarElement is incorrect since HTMLElements are the only ones that can have focus.

Reply:
what about `SugarElement<HTMLELement>`?

Reply:
changed to `SugarElement<HTMLELement>`